src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -123,17 +129,22 @@ class ReportActionContextMenu extends React.Component {
                     );
                 },
             },
-
             {
                 text: this.props.translate('reportActionContextMenu.deleteComment'),
                 icon: Trashcan,
-                shouldShow: false,
-                onPress: () => {},
+                shouldShow: () => this.props.reportAction.actorEmail === this.props.session.email,
+                onPress: () => {
```

Comment:
No need for brackets here:
```suggestion
                onPress: () => this.setState({isDeleteCommentConfirmModal: true}),
```